cpp/arcticdb/version/python_bindings.cpp

```diff
@@ -130,6 +132,20 @@ void register_bindings(py::module &version, py::exception<arcticdb::ArcticExcept
         .def_property_readonly("names", &PythonOutputFrame::names, py::return_value_policy::reference)
         .def_property_readonly("index_columns", &PythonOutputFrame::index_columns, py::return_value_policy::reference);
 
+    py::enum_<VersionRequestType>(version, "VersionRequestType")
+            .value("SNAPSHOT", VersionRequestType::SNAPSHOT)
+            .value("TIMESTAMP", VersionRequestType::TIMESTAMP)
+            .value("SPECIFIC", VersionRequestType::SPECIFIC)
+            .value("LATEST", VersionRequestType::LATEST);
+
+    py::class_<DataError, std::shared_ptr<DataError>>(version, "DataError")
+            .def_property_readonly("symbol", &DataError::symbol)
+            .def_property_readonly("version_request_type", &DataError::version_request_type)
+            .def_property_readonly("version_request_data", &DataError::version_request_data)
+            .def_property_readonly("error_code", &DataError::error_code)
+            .def_property_readonly("error_category", &DataError::error_category)
+            .def_property_readonly("exception_string", &DataError::exception_string)
+            .def("__repr__", &DataError::to_string);
```

Comment:
This isn't a `__repr__`. `__repr__` would print something like `DataError(error_code=..., error_category=...)`.

I think this should be `__str__`.

Reply:
Changed